Apps/Admin/Tests/Functional/cypress/integration/e2e/analytics.cy.js

```diff
@@ -18,10 +20,9 @@ describe("System Analytics", () => {
 
         cy.intercept("GET", "**/CsvExport/GetRatings").as("getRatings");
 
-        cy.intercept(
-            "GET",
-            "**/CsvExport/GetInactiveUsers?timeOffset=420&inactiveDays=100"
-        ).as("getInactiveUsers");
+        cy.intercept("GET", "**/CsvExport/GetInactiveUsers*").as(
```

Comment:
Reason why offset and inactiveDays were specified was to ensure that those values were being read in the intercept vs just using *.

Reply:
It seems like it should work with the parameters specified, and it does locally, but it fails when running in CI with this error:

> Timed out retrying after 5000ms: cy.wait() timed out waiting 5000ms for the 1st request to the route: getInactiveUsers. No request ever occurred.

I thought perhaps the parameters were different in dev than locally (due to time zones), which is why I removed them from the intercept.  After checking dev, this doesn't seem to be true, but I'm not sure why else it would be failing.

Reply:
I think you're right.  Hard-coding 420 is not good as that can change with Daylight Savings Time and Timezone of where this is being run.

Reply:
Okay, I'll leave it as it is in the PR for now and we can see if that fixes the CI runs or if something else is at play.